File: FindtheHighestAltitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        
        int n = gain.length;
        int res = 0;
        int mx = 0;
        for(int i = 0;i < gain.length;i++){
            res += gain[i];
            mx = Math.max(res, mx);
        }

        return mx;
    }
}

File: FindtheHighestAltitude.cs
public class Solution {
    public int LargestAltitude(int[] gain) {
        int max = 0;
        int len = gain.Length;
        int curAltitude = 0;
        for(int i = 0;i < len;i++){
            curAltitude += gain[i];
            max = Math.Max(curAltitude, max);
        }
        return max;
    }
}
